### DISCUSSION TITLE
No defaultFoo props [solved]

### DISCUSSION THREAD

**Discussion by yannxaver** [visgl/react-google-maps#202] [General]
> BREAKING CHANGES
> 
> The behaviour of the props controlling camera parameters (center, zoom, heading and tilt) changed. Unless you are using > controlled props, you have to change the prop names from e.g. center to defaultCenter (the same goes for zoom, heading and tilt).

I upgraded to 0.6.0 and I cannot find these default props.

**Comment by usefulthink** [comment]
What do you mean, you can't find them? Just had a look through the installed package files and it's both in the `.js` as well as in the `.d.ts` and `.ts` files.

**Reply by yannxaver** [reply]
I had to restart vsCode a couple of times and then they showed up in IntelliSense. Yesterday, before writing the post, I restarted the editor just once and they still didn't show up. I cannot remember having this problem before, but probably a vsCode regression.
